packages/contracts/contracts/test/CommonTest.t.sol

```diff
@@ -595,9 +634,10 @@ contract FFIInterface is Test {
         return abi.decode(result, (bytes32));
     }
 
-    function encodeDepositTransaction(
-        Types.UserDepositTransaction calldata txn
-    ) external returns (bytes memory) {
+    function encodeDepositTransaction(Types.UserDepositTransaction calldata txn)
+        external
+        returns (bytes memory)
+    {
```

Comment:
Could you restore to the original code? The original code is the right code which conforms to formatting rule.